main.go

```diff
@@ -25,17 +28,16 @@ import (
 var maxVersion = 10000
 
 // Settings for daemon
-var dcrdCertPath = ("/home/user/.dcrd/rpc.cert")
-var dcrdServer = "127.0.0.1:19109"
-var dcrdUser = "USER"
-var dcrdPass = "PASSWORD"
+var host = flag.String("host", "127.0.0.1:19109", "node RPC host:port")
+var user = flag.String("user", "dcrd", "node RPC username")
+var pass = flag.String("pass", "bananas", "node RPC password")
+var cert = flag.String("cert", "dcrd.cert", "node RPC TLS certificate (when notls=false)")
+var notls = flag.Bool("notls", true, "Disable use of TLS for node connection")
```

Comment:
With this bool default to true, I can't 'not' use notls, so probably should instead put it to default to false for notls then flip to true on flag being seen on cli